tests/unit/helpers/mocks.py

```diff
@@ -54,8 +54,9 @@ def __init__(self, *args, **kwargs):
 
 
 class MockFiles(object):
-    def __init__(self, data):
+    def __init__(self, data, mimetype='application/pdf'):
```

Comment:
This additional parameter is hack to personalize the behavior of the whole hierarchy.